aspnetcore/blazor/common/samples/3.x/BlazorSample/Pages/BlazorRocksBase.cs

```diff
@@ -1,7 +1,9 @@
+/* XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 using Microsoft.AspNetCore.Components;
 
 public class BlazorRocksBase : ComponentBase
```

Comment:
When I tried putting this type in a namespace (specifically BlazorSample) it worked.